### PR TITLE
chore: update local-celestia-devnet version

### PR DESCRIPTION
**Pull request by jcstein**
## Checklist

- [ ] New and updated code has appropriate documentation
- [ ] New and updated code has new and/or updated testing
- [ ] Required CI checks are passing
- [ ] Visual proof for any user facing features like CLI or documentation updates
- [ ] Linked issues closed with keywords

### CODE REVIEW

**Review by tuxcanfly — APPROVED** [review]
OK.

File: ops-bedrock/docker-compose.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ volumes:
 services:
   da:
     platform: linux/x86_64
-    image: "ghcr.io/rollkit/local-celestia-devnet:v0.11.0"
+    image: "ghcr.io/rollkit/local-celestia-devnet:v0.12.1"
```
Comment (tuxcanfly) [inline ops-bedrock/docker-compose.yml:16]: We should `docker-compose-testnet.yml` while we are at it.

Comment (jcstein) [reply ops-bedrock/docker-compose.yml:16]: 🫡 https://github.com/celestiaorg/optimism/commit/ca7ed14a4a1593e2109960a4b3d07e5a208e0a6b

Comment (jcstein) [reply ops-bedrock/docker-compose.yml:16]: utACK though

Comment (jcstein) [reply ops-bedrock/docker-compose.yml:16]: note: used v0.12.0 for celestia-node, as the local-celestia-devnet v0.12.1 release contains a patch fix that is not relevant to celestia-node

**Pull request merged**